fvm/errors/contracts.go

```diff
@@ -33,28 +33,3 @@ func (e *ContractNotFoundError) Error() string {
 func (e *ContractNotFoundError) Code() ErrorCode {
 	return ErrCodeContractNotFoundError
 }
-
-// ContractNamesNotFoundError is returned when fetching a list of contract names under an account
-type ContractNamesNotFoundError struct {
-	address flow.Address
-}
-
```

Comment:
hmm, shall we return this when its happening? what we do right now in that case?